zcash_client_backend/src/data_api.rs

```diff
@@ -1698,6 +1698,31 @@ pub trait WalletRead {
         unimplemented!("WalletRead::get_transparent_receivers must be overridden for wallets to use the `transparent-inputs` feature")
     }
 
+    /// Returns the set of previously-exposed ephemeral transparent receivers generated by the given
+    /// account controlled by this wallet.
+    ///
+    /// The set contains all ephemeral transparent receivers that are known to have been derived
+    /// under this account within `exposure_depth` blocks of the chain tip. Wallets may scan the
+    /// chain for UTXOs sent to these receivers, but should do so in a fashion that does not reveal
+    /// that they are controlled by the same wallet.
+    ///
+    /// # Parameters
+    /// - `account`: The identifier for the account from which transparent receivers should be
+    ///   returned.
+    /// - `exposure_depth`: Implementations of this method should return only addresses exposed at
+    ///   heights greater than or equal to `chain_tip_height - exposure_depth`
+    /// - `exclude_used`: When set to `true`, do not return addresses that are known have already
+    ///   received funds in a transaction.
+    #[cfg(feature = "transparent-inputs")]
+    fn get_ephemeral_transparent_receivers(
```

Comment:
How does this differ in intention, usage, or returned data from `get_known_ephemeral_addresses`? The documentation for this method does not reference the other, and the documentation for the other method is not updated to account for the presence of this method. Both methods need sufficient documentation for callers to understand how to use them both correctly, and why there are two.